src/manifest_parser.h

```diff
@@ -15,10 +15,16 @@
 #ifndef NINJA_MANIFEST_PARSER_H_
 #define NINJA_MANIFEST_PARSER_H_
 
+#include <string>
+#include <vector>
+using namespace std;
```

Comment:
Please do not add "using namespace std;" to any files, and for new or modified code, add "std::" where appropriate.